roles/openshift_logging/templates/elasticsearch-logging.yml.j2

```diff
@@ -28,13 +39,14 @@ logger:
 additivity:
   index.search.slowlog: false
   index.indexing.slowlog: false
+  deprecation: false
 
 appender:
```

Comment:
I don't think so.  The appenders are static blocks that will only be used when a logger is told to use them, unless I am missing something.